### DISCUSSION TITLE
Loading localResource html within iframe and webview has stopped working after 1.56 version

### DISCUSSION THREAD

**Discussion by Uma19** [microsoft/vscode-discussions#12] [Extension Development QnA] [answered]
Hello Team,
Loading localResource html within iframe and binding to webview was working with 1.55 version of vscode but now after 1.56 it has stopped working.

Do we have any alternative to the same ?

**Comment by isidorn** [comment, marked as answer]
This sounds like a VS Code regression. I suggest that you open a new issue in the VS Code repository.
Also make sure you can still reproduce with the latest VS Code version. 1.56 is quite old

**Reply by Uma19** [reply]
Thanks Isidorn.
Let me raise a bug with the team
